src/kudu/consensus/raft_consensus.cc

```diff
@@ -511,9 +515,15 @@ Status RaftConsensus::StartElection(ElectionMode mode, ElectionReason reason) {
                                    << SecureShortDebugString(active_config);
 
     // Initialize the VoteCounter.
-    int num_voters = CountVoters(active_config);
-    int majority_size = MajoritySize(num_voters);
-    gscoped_ptr<VoteCounter> counter(new VoteCounter(num_voters, majority_size));
+    gscoped_ptr<VoteCounter> counter;
+
+    if (!FLAGS_enable_flexi_raft) {
+      int num_voters = CountVoters(active_config);
+      int majority_size = MajoritySize(num_voters);
+      counter.reset(new VoteCounter(num_voters, majority_size));
+    } else {
+      counter.reset(new FlexibleVoteCounter(active_config));
```

Comment:
in Leader Election, you seem to be tabulating whether the Data Quorum is VOTE_GRANTED.
You will need to do the dual of that.

Reply:
Renamed as per our conversation.